### DISCUSSION TITLE
Quire Resource from the Denver Art Museum

### DISCUSSION THREAD

**Discussion by Erin-Cecele** [thegetty/quire#961] [Tips, Tricks, & Resources]
Valerie Hellstein ([@vhellstein](https://github.com/vhellstein/)), the Managing Editor for Publications at the [Denver Art Museum](https://www.denverartmuseum.org/en), put together this fantastic resource to share with her curatorial colleagues. This slide deck outlines why the DAM publishes digital catalogues, who the audiences are, and how the catalogues are promoted. It also explains what Quire is, why they use it, and how it works, in addition to breaking down team roles and workflow. 

We highly recommend perusing this resource, especially if you need to make the case for digital publishing or Quire at your own institution.

[Denver_Art_Museum_Quire_Deck.pdf](https://github.com/user-attachments/files/17331358/Denver_Art_Museum_Quire_Deck.pdf)

![Digital_Publishing_at_DAM](https://github.com/user-attachments/assets/0d55138e-9bb5-47c7-bf46-97011b70ebf5)

So far, the Denver Art Museum has two publications with Quire: [Biophilia](https://biophilia.denartmus.org/) (2024) and [Perfectly Imperfect](https://buncheong.denartmus.org/) (2023). They have several more in the pipeline.

**Comment by tiffanysprague** [comment]
SUPER helpful, thank you for sharing! Congrats on your lovely digital catalogues @vhellstein! 
--
Tiffany Sprague
Director of Publications and Editorial Services
Yale University Art Gallery

**Comment by adhyan1991-hue** [comment]
This is an incredibly valuable resource thank you for sharing! It’s really helpful to see how the Denver Art Museum approaches digital publishing, from defining audiences to structuring workflows and team roles. The insights into how Quire is implemented in real-world projects make it much easier to understand its practical benefits. I also find it interesting how institutions are increasingly connecting digital publications with broader digital experiences, such as integrating access through a [digital membership card](https://membershipanywhere.com/digital-membership/) which can help improve audience engagement and accessibility. The examples of Biophilia (2024) and Perfectly Imperfect (2023) are especially inspiring, and it’s great to hear that more publications are in progress. Resources like this make a strong case for adopting digital catalogues and tools like Quire in other institutions as well.
